projects/demo/src/modules/app/app.routes.ts

```diff
@@ -120,6 +120,30 @@ export const ROUTES = [
             title: `Dialogs`,
         },
     },
+    {
+        path: `navigation/generate-dialogable-route`,
+        loadChildren: async () =>
+            (
+                await import(
+                    `../customization/dialogable-route/eager/dialogable-route.module`
+                )
+            ).DialogableRouteModule,
+        data: {
+            title: `GenerateDialogableRoute`,
+        },
+    },
+    {
+        path: `navigation/generate-lazy-dialogable-route`,
+        loadChildren: async () =>
+            (
+                await import(
+                    `../customization/dialogable-route/lazy/lazy-dialogable-route.module`
```

Comment:
Is `customization` good folder for demo?

Reply:
I think it's best to group

**Before**

<img width="277" alt="image" src="https://user-images.githubusercontent.com/12021443/218568714-c32b6313-0505-415a-a4fb-ca2be577e92e.png">

**After**

<img width="201" alt="image" src="https://user-images.githubusercontent.com/12021443/218569005-b62fa8d3-d379-47ec-aeca-9a9737b82574.png">

Reply:
Done